server/server.go

```diff
@@ -1805,6 +1808,10 @@ func RunServerWithOpts(stanOpts *Options, natsOpts *server.Options) (newServer *
 	return &s, nil
 }
 
+func (s *StanServer) ClientURL() string {
```

Comment:
Since this is exported, should have a comment:
```
// ClientURL returns the basic URL string representation suitable for a client to use to connect
```
But see the comment about when the NATS server is not embedded.